catalogd/Makefile

```diff
@@ -98,18 +88,15 @@ export GO_BUILD_TAGS     := containers_image_openpgp
 
 BUILDCMD = go build -tags '$(GO_BUILD_TAGS)' -ldflags '$(GO_BUILD_LDFLAGS)' -gcflags '$(GO_BUILD_GCFLAGS)' -asmflags '$(GO_BUILD_ASMFLAGS)' -o $(BUILDBIN)/$(notdir $@) ./cmd/$(notdir $@)
 
-.PHONY: build-deps
-build-deps: generate
-
 .PHONY: build go-build-local $(BINARIES)
-build: build-deps go-build-local ## Build binaries for current GOOS and GOARCH.
+build: go-build-local ## Build binaries for current GOOS and GOARCH.
```

Comment:
This is a little bit unfortunate, as it requires that the root Makefile has already run it's `generate` target. But since this is just temporary, and we don't have any CI or dev flows where we would be running `make` tasks directly in the `catalogd` Makefile, I think its fine.

Reply:
Agree. We should move rest of the Makefile soon and that would remove the gap.